pkg/reconciler/knservice/knservice_reconciler_test.go

```diff
@@ -0,0 +1,130 @@
+package knservice
+
+import (
+	"context"
+	"github.com/google/go-cmp/cmp"
+	knservingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"github.com/kubeflow/kfserving/pkg/containers/tensorflow"
+	"github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestKnativeServiceReconcile(t *testing.T) {
+	existingService := &knservingv1alpha1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "mnist",
+			Namespace: "default",
+		},
```

Comment:
Let's wire through all Labels and Annotations onto subresources. This will make them both configurable and recognizable.